tensorflow/contrib/lite/examples/python/label_image.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""label_image for tflite"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import argparse
+import numpy as np
+
+from PIL import Image
+
+from tensorflow.contrib.lite.python import interpreter as interpreter_wrapper
+
+def load_labels(filename):
+  my_labels = []
+  input_file = open(filename, 'r')
+  for l in input_file:
+    my_labels.append(l.strip())
+  return my_labels
+
+if __name__ == "__main__":
+  file_name = "/tmp/grace_hopper.bmp"
+  model_file = "/tmp/mobilenet_v1_1.0_224_quant.tflite"
+  label_file = "/tmp/labels.txt"
+  input_mean = 127.5
+  input_std = 127.5
+  floating_model = False
+
+  parser = argparse.ArgumentParser()
+  parser.add_argument("--image", help="image to be classified")
+  parser.add_argument("--graph", help=".tflite model to be executed")
```

Comment:
nit: Use `model_file`, `tflite_file`, or `tflite_model_file` instead of `graph`.

Reply:
done